deploy/template.yaml

```diff
@@ -30,6 +30,8 @@ Resources:
     Properties:
       Name: IPV CRI UK Passport API Gateway
       StageName: !Sub ${Environment}
+      Auth:
+        ApiKeyRequired: 'true'
```

Comment:
Should this be without quotes? The docs say it's a boolean. Not sure how YAML or Cloudformation interprets booleans